src/components/ShowCard.tsx

```diff
@@ -1,19 +1,50 @@
+import { MovieDetailsData } from '../types/tmdb';
+interface MovieCardProps { details: MovieDetailsData | null }
 /**
  * Show cards are rendered all over the application in different situations
  * Be sure changes made to this component are either conditionally applied
  * or intended to be on every single show card
  * 
  * @returns {JSX.Element} | Single show card
  */
-export default function ShowCard(): JSX.Element {
+export default function ShowCard(props: MovieCardProps) {
+	console.log(props);
+	const ratingHandler = (arr: any) => {
+		const newArr: any[] = [];
```

Comment:
Can we use the ReleaseDate type here instead of `any` ?

Reply:
For this `arr` and filter. Is there a reason we are using an array? Can we just check if the details contains a US release date and then apply that rating? Or is this set up to allow for more dynamic code in the future?
If we could remove the array, we would also remove the `any` type by default.

Reply:
@Thenlie Initial thought was with multiple languages/country filters pushed to an array and easily selectable. Can remove if needed.